modules/core/test/v2/unit/coins/ltc.ts

```diff
@@ -63,21 +63,35 @@ describe('LTC:', function() {
       version2Address.should.equal(bech32Address);
     });
 
-    it('upper case bech32 mainnet address', function() {
+    it('upper case bech32 mainnet address should fail', function () {
       // bech32 addresses which are all upper case or all lower case are potentially valid,
       // but mixed case is always invalid. Canonical bech32 addresses should be all lower case however
       const bech32Address = 'LTC1QGRL8ZPNDSKLAA9SWGD5VEVYXMX5X63VCRL7DK4';
-      const newAddress = ltc.canonicalAddress(bech32Address);
-      newAddress.should.equal(bech32Address.toLowerCase());
+      try {
+        ltc.canonicalAddress(bech32Address);
+        should.fail(null, null, 'Upper case bech32 addresses are not valid');
+      } catch (e) {
+        e.message.should.equal('invalid address');
+      }
```

Comment:
This is a more idiomatic way to make sure a function throws a given error string.

```suggestion
      (() => ltc.canonicalAddress(bech32Address)).should.throw('invalid address');
```

Reply:
Suggestion applied